### PR TITLE
CLOUDSTACK-8645: Improve logging of RBD functionality in KVM

### PR DESCRIPTION
**Pull request by wido**
A simple commit which changes a couple of log lines.

### CODE REVIEW

**Comment by borisroman** [comment]
Pending Travis -> LGTM

File: plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/storage/LibvirtStorageAdaptor.java [hunk 1]
```diff
@@ -877,8 +877,8 @@ public boolean deletePhysicalDisk(String uuid, KVMStoragePool pool, Storage.Imag
                 rbd.close(image);
                 r.ioCtxDestroy(io);
 
-                s_logger.debug("Succesfully unprotected and removed any snapshots of " + pool.getSourceDir() + "/" + uuid +
-                        " Continuing to remove the RBD image");
+                s_logger.info("Succesfully unprotected and removed all snapshots " + snaps.size() + " of " + pool.getSourceDir() + "/" + uuid +
```
Comment (DaanHoogland) [inline plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/storage/LibvirtStorageAdaptor.java]: This is going to read something like "Succesfully unprotected and removed all snapshots 5 of /<somepath>/<some uuid> Continuing to remove the RBD image", in the logs. the part "all snapshots # of ..." seems not what you'd want to read. How about "Succesfully unprotected and removed any remaining snapshots (counting " + snaps.size() + ") of " + ...?

**Comment by DaanHoogland** [comment]
comment about the log message, otherwise LGTM

**Comment by wido** [comment]
@DaanHoogland Pushed a fix for that log line

**Comment by borisroman** [comment]
@wido Can you squash both commits into one?

**Comment by wido** [comment]
@borisroman Done!

**Comment by borisroman** [comment]
LGTM

**Pull request merged**